colossalai/zero/shard_utils/base_shard_strategy.py

```diff
@@ -7,21 +7,30 @@
 class BaseShardStrategy(ABC):
 
     def __init__(self, process_group: Optional[dist.ProcessGroup] = None) -> None:
+        """Abstract Shard Strategy. Use to shard a tensors on multiple GPUs.
+
+        Args:
+            process_group (Optional[dist.ProcessGroup], optional): the process group. Defaults to None.
+        """
```

Comment:
This format is not consistent with the current sphinx docstring style, thus, ReadTheDocs will not be able to parse this docstring.

Reply:
delete comments

Reply:
got it